aii-core/src/main/perl/DHCP.pm

```diff
@@ -495,6 +495,38 @@ sub read_input
     return $error;
 }
 
+# update the dhcp config file and restart daemon
+sub update_and_restart {
+	my $self = shift;
+
+    my $filename = $self->option('dhcpconf');
+    $self->debug(1, "aii-dhcp: going to update dhcpd configuration $filename");
```

Comment:
the `aii-dhcp` is not correct. i would also not report `update` since you don't know yet if there will be a change or not.